### PR TITLE
PR for issue #38

### PR DESCRIPTION
**Pull request by mpatsute**
1. Generated datasource mongoDS
2. Generated model Cheque to talk to mongo DB
3. Updated docker-compose to dockerify mongo

### CODE REVIEW

**Comment by superkhau** [comment]
Please use a more descriptive commit message -- see http://loopback.io/doc/en/contrib/git-commit-messages.html.

File: services/transaction/server/datasources.json [hunk 1]
```diff
@@ -1,7 +1,12 @@
 {
   "TransactionDB": {
-    "name": "db",
+    "name": "TransactionDB",
     "connector": "memory",
     "file": "data.json"
+  },
+  "mongoDs": {
```
Comment (superkhau) [inline services/transaction/server/datasources.json]: Maybe name it something more obvious? mongoService?

File: services/transaction/server/model-config.json [hunk 2]
```diff
@@ -20,5 +20,9 @@
   "Vital": {
     "dataSource": null,
     "public": true
+  },
+  "Cheque": {
+    "dataSource": "mongoDs",
```
Comment (superkhau) [inline services/transaction/server/model-config.json]: mongoService

**Review by superkhau — CHANGES_REQUESTED** [review]
One nitpick, LGTM.

File: services/transaction/server/datasources.json (hunk 1 above)
Comment (mpatsute) [reply services/transaction/server/datasources.json]: Renamed mongoDs to mongoService!

**Review by superkhau — APPROVED** [review]

**Comment by superkhau** [comment]
@mpatsute You should squash the commits into one and use a more descriptive commit message like:

"Use MongoDB in the Transaction microservice".

**Comment by ritch** [comment]
LGTM

**Pull request merged**